### DISCUSSION TITLE
DomPDF Support

### DISCUSSION THREAD

**Discussion by ejaz-ul-haq** [twbs/bootstrap#31905] [Ideas]
Hi, 
I am Ejaz WordPress & Laravel Developer.

Bootstrap is an efficient FrontEnd development framework but
Developer communities are facing issues in V4 regarding PDF libraries support
like DomPDF is widely used By Laravel and WordPress Developers with Bootstrap 3.3.6 But
But Bootstrap V4 is not comfortable anymore.
I hope this issue will be resolved in V5, that we can enjoy Bootstrap everywhere. 

Thanks.

**Comment by alecpl** [comment]
You don't expect Bootstrap community to work on DomPDF, do you? If not, then what would you expect?

**Reply by ejaz-ul-haq** [reply]
I am expecting that Bootstrap V5 should work with PDF libraries like Bootstrap V3.3.6. Thanks

**Comment by patrickhlauke** [comment]
moving this to a discussion. it would be good to elaborate a bit more what exactly it is that makes Bootstrap "not comfortable anymore" with PDF libraries, rather than a generic "make it work"

**Reply by ejaz-ul-haq** [reply]
Yes, if the Bootstrap community takes an effort to resolve the CSS issues with Print so it will be a graceful collaboration for Backend dev communities.

**Reply by patrickhlauke** [reply]
again, if you could actually be more specific on what currently does/doesn't work, rather than just a blanket "resolve the CSS issues" ... i.e. list the problems you're seeing, at least

**Comment by alecpl** [comment]
I guess the main issue here is that DomPDF does not support flexbox.
